src/Html/Attributes.php

```diff
@@ -8,6 +8,8 @@ class Attributes
 
     protected array $classes = [];
 
+    protected array $ids = [];
```

Comment:
An element can only have one ID.
```suggestion
    protected ?string $id = null;
```